test/integration/userStatus.test.js

```diff
@@ -554,6 +556,135 @@ describe("UserStatus", function () {
     });
   });
 
+  describe("PATCH /users/status/self", function () {
+    let userId;
+    let userJwt;
+
+    before(async function () {
+      userId = await addUser(userData[8]);
+      userJwt = authService.generateAuthToken({ userId });
+    });
+
+    afterEach(async function () {
+      await firestore.collection("tasks").doc("user1AssignedTask").delete();
+      await firestore.collection("usersStatus").doc("user1AssignedStatus").delete();
+    });
+
+    after(async function () {
+      await firestore.collection("users").doc(userId).delete();
+    });
+
+    it("Should Change the status to ACTIVE if user has task assigned.", async function () {
+      const now = new Date();
+      const nowTimeStamp = new Date().setUTCHours(0, 0, 0, 0);
+      const fiveDaysFromNowTimeStamp = new Date(now.setUTCHours(0, 0, 0, 0) + 5 * 24 * 60 * 60 * 1000);
+
+      await firestore.collection("tasks").doc("user1AssignedTask").set({
+        assignee: userId,
+        status: "ASSIGNED",
+      });
```

Comment:
Nit: For learning purpose I am asking 
we are using production firestore for adding data??

If you redirect me to any docs : want to read about this